ftw/iframeblock/profiles/default_plone5/propertiestool.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
```

Comment:
This does not work in plone5. Translate/Move to the equivalent `registry.xml` records.

Reply:
👍 I added the resources to registry.xml